### PR TITLE
refactor(evs): refactor api version of evs volume from v3 to v2

### LINKED ISSUE #1817: [EVS] Use new V2 API to refactor evs resources
We need to refactor the evs resources with new V2 API due to V3 API deprecated and OpenStack API not recommended.

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
- The API version is changed from V3 to V2.1, and the function and parameter support remain unchanged.
- For about acc test:
  - Combine the tags and image test into the basic test

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #1817 

**Special notes for your reviewer**:

**Release note**:

```release-note
1. refactor api version form v3 to v2.
```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/evs' TESTARGS='-run=TestAccEvsVolume_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/evs -v -run=TestAccEvsVolume_basic -timeout 360m -parallel 4
=== RUN   TestAccEvsVolume_basic
=== PAUSE TestAccEvsVolume_basic
=== CONT  TestAccEvsVolume_basic
--- PASS: TestAccEvsVolume_basic (149.56s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/evs       149.991s
```
```
make testacc TEST='./huaweicloud/services/acceptance/evs' TESTARGS='-run=TestAccEvsVolume_withEpsId'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/evs -v -run=TestAccEvsVolume_withEpsId -timeout 360m -parallel 4
=== RUN   TestAccEvsVolume_withEpsId
=== PAUSE TestAccEvsVolume_withEpsId
=== CONT  TestAccEvsVolume_withEpsId
--- PASS: TestAccEvsVolume_withEpsId (91.62s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/evs       91.765s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/evs/resource_huaweicloud_evs_volume_test.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+	resourceName := "huaweicloud_evs_volume.test"
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&volume,
+		getVolumeResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
```
Comment (hmbig2) [inline huaweicloud/services/acceptance/evs/resource_huaweicloud_evs_volume_test.go]: please add eps precheck

**Comment by Lance52259** [comment]
```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccEvsSnapshotV2_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccEvsSnapshotV2_basic -timeout 360m -parallel 4
=== RUN   TestAccEvsSnapshotV2_basic
=== PAUSE TestAccEvsSnapshotV2_basic
=== CONT  TestAccEvsSnapshotV2_basic
--- PASS: TestAccEvsSnapshotV2_basic (90.50s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       90.683s
```
```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccCCIPersistentVolumeClaims_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run=TestAccCCIPersistentVolumeClaims_basic -timeout 360m -parallel 4
=== RUN   TestAccCCIPersistentVolumeClaims_basic
=== PAUSE TestAccCCIPersistentVolumeClaims_basic
=== CONT  TestAccCCIPersistentVolumeClaims_basic
--- PASS: TestAccCCIPersistentVolumeClaims_basic (105.19s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       105.558s
```
```
make testacc TEST='./huaweicloud/services/acceptance/evs' TESTARGS='-run=TestAccEvsVolume_withEpsId'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/evs -v -run=TestAccEvsVolume_withEpsId -timeout 360m -parallel 4
=== RUN   TestAccEvsVolume_withEpsId
=== PAUSE TestAccEvsVolume_withEpsId
=== CONT  TestAccEvsVolume_withEpsId
--- PASS: TestAccEvsVolume_withEpsId (91.33s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/evs       91.677s
```

File: huaweicloud/services/acceptance/evs/resource_huaweicloud_evs_volume_test.go (hunk 1 above)
Comment (Lance52259) [reply huaweicloud/services/acceptance/evs/resource_huaweicloud_evs_volume_test.go]: sure

Comment (Lance52259) [reply huaweicloud/services/acceptance/evs/resource_huaweicloud_evs_volume_test.go]: sure

File: huaweicloud/services/evs/resource_huaweicloud_evs_volume.go [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+	opt := buildEvsVolumeCreateOpts(d, config)
+	logp.Printf("[DEBUG] Create Options: %#v", opt)
+	job, err := cloudvolumes.Create(evsV21Client, opt).Extract()
+	if err != nil {
+		return fmtp.DiagErrorf("Error creating HuaweiCloud EVS volume: %s", err)
+	}
+	logp.Printf("[INFO] Volume Job ID: %s", job.JobID)
+
+	// Waiting for the volume to become available.
+	logp.Printf("[DEBUG] Waiting for volume to become available.")
```
Comment (hmbig2) [inline huaweicloud/services/evs/resource_huaweicloud_evs_volume.go]: pls add some ID that is useful to debug

Comment (Lance52259) [reply huaweicloud/services/evs/resource_huaweicloud_evs_volume.go]: done

**Pull request merged**